.github/workflows/release.yml

```diff
@@ -0,0 +1,69 @@
+name: Create release
+on:
+  workflow_dispatch:
+
+jobs:
+  get-app-version:
+    runs-on: ubuntu-latest
+    outputs:
+      APP_VERSION: ${{ steps.get-application-verion.outputs.APP_VERSION }}
+    steps:
+    - id: get-application-verion
+      name: get application version
+      run: |
+        APP_VERSION=$(date +"%Y.%m.%d")
```

Comment:
Why we get app version from `makefile` in case of builds, but here it seems to be just current date? Why not `APP_VERSION=$(make get-app-version)`?

Reply:
In `Makefile` its
```
$(shell date +"%Y.%m.%d")-$(shell git rev-parse --short HEAD)
```
so it also cotains suffix of short git hash. We most likely don't want to include that in releases published for our customers.
I agree we should most likely encapsulate this logic also in `Makefile` (maybe having `DEV_APP_VERSION` and `RELEASE_APP_VERSION`), but I decided in this pull request to leave `Makefile` as it is (more or less at least).
Let's have `Makefile` cleanup done in separate pull request (maybe there will be also room to include instruction how to run Makefile on Windows? an maybe how to setup dev environmetn on Windows).